nextflow/nextflow.config

```diff
@@ -179,7 +125,10 @@ else if ( hostname.startsWith("farnarkle") ) {
     params.presto_module = 'd6265c2'
     params.basedir = '/fred/oz125/vcs'
     params.scratch_basedir = '/fred/oz125/vcs'
-    params.search_dir = '/fred/oz125/nswainst/pulsar_search'
+    params.search_dir = '/fred/oz125/${USER}/pulsar_search'
+
+    params.vcstools_version = 'master'
+    params.mwa_search_version = 'master'
```

Comment:
You also need this for x86 (the shanghai server)

Reply:
Okay, done.